chart/validator/templates/cleanup.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: spectro-cleanup-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - serviceaccounts
+  verbs:
+  - '*'
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - '*'
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  - roles
+  verbs:
+  - '*'
+- apiGroups:
+  - validation.spectrocloud.labs
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: spectro-cleanup-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: spectro-cleanup-role
+subjects:
+  - kind: ServiceAccount
+    name: spectro-cleanup
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spectro-cleanup
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: validator-cleanup-config
+data:
+  resource-config.json: |-
+    [
+      {
+        "group": "validation.spectrocloud.labs",
+        "version": "v1alpha1",
+        "resource": "validatorconfigs",
+        "name": "validator-config",
+        "namespace": "{{ .Release.Namespace }}"
+      },
+      {
+        "group": "",
+        "version": "v1",
+        "resource": "configmaps",
+        "name": "validator-cleanup-config",
+        "namespace": "{{ .Release.Namespace }}"
+      },
+      {
+        "group": "batch",
+        "version": "v1",
+        "resource": "jobs",
+        "name": "validator-cleanup",
+        "namespace": "{{ .Release.Namespace }}"
+      }
+    ]
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: validator-cleanup
+  annotations:
+    "helm.sh/hook": pre-delete
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: spectro-cleanup
+      containers:
+      - name: validator-cleanup
+        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.1.0
```

Comment:
I wanted to check out the code for spectro-cleanup to get some more context. Where would I find that?

Reply:
@mattwelke https://github.com/spectrocloud/utils/blob/main/spectro-cleanup/main.go